src/screens/Connections/Connections.tsx

```diff
@@ -22,7 +22,11 @@ import EmptyState from './components/EmptyState/EmptyState';
 import Switch from './components/Switch/Switch';
 import {flights} from '../../data';
 
-const Connections = () => {
+type Props = {
+  navigation: any;
```

Comment:
Replace navigation: `any` with navigation:`NavigationProp<RootStackParamsList, 'Connections'>;`

Import `NavigationProp` from '@react-navigation/native'. With this it's easier to get navigation methods and also easy lists of screen names from `RootStackParamsList`